pkg/operator/ceph/cluster/controller.go

```diff
@@ -759,14 +760,22 @@ func (c *ClusterController) onDelete(obj interface{}) {
 }
 
 func (c *ClusterController) handleDelete(cluster *cephv1.CephCluster, retryInterval time.Duration) error {
-
+	flexDriverEnabled := os.Getenv(enableFlexDriver) == "true"
```

Comment:
Upgraded clusters will actually not have this property, but it will be enabled by default during upgrades. So I believe we need this condition:
```
os.Getenv(enableFlexDriver) != "false"
```

Reply:
Ah, I see.